### PR TITLE
update-delete-or-restore-mailbox-task4455945

### PR DESCRIPTION
**Pull request by Lovina-Saldanha**
Removed section "how do you know this worked" and moved "Use Exchange Online PowerShell" to the end in each topic.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/delete-or-restore-mailboxes.md [hunk 1]
```diff
@@ -85,25 +71,19 @@ Remove-MsolUser -UserPrincipalName <Walter Harp> -RemoveFromRecycleBin
...
-
-To verify that you've successfully deleted an Exchange Online mailbox, do one of following:
-
-- In the EAC, navigate to **Recipients** \> **Mailboxes**. The deleted mailbox is removed from the mailbox list.
-
-  Click **Refresh** ![Refresh Icon](../media/ITPro_EAC_RefreshIcon.gif) if the deleted mailbox is still displayed.
+### Use Exchange Online PowerShell to delete a mailbox
 
-- If you deleted the user account, verify that the user account isn't listed on the **Active users** page in the Microsoft 365 admin center, and that it's listed on the **Deleted Users** page.
+- You need to be assigned permissions before you can perform this procedure or procedures. To see what permissions you need, see the"Recipients" entry in the [Feature permissions in Exchange Online](../permissions-exo/feature-permissions.md) topic.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/delete-or-restore-mailboxes.md:76]:
```suggestion
- You need to be assigned permissions before you can perform this procedure or procedures. To see what permissions you need, see the "Recipients" entry in the [Feature permissions in Exchange Online](../permissions-exo/feature-permissions.md) topic.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**